docs/docs/genai/serving/index.mdx

```diff
@@ -56,237 +56,428 @@ mlflow models serve -m "models:/<model-name>/<model-version>" -p 5000
-    MODEL_B_URL = "http://localhost:5001/invocations"  # New model
-
-    # Traffic split configuration
-    TRAFFIC_SPLIT = {
-        "model_a": 0.8,  # 80% to current model
-        "model_b": 0.2,  # 20% to new model
-    }
+</TabItem>
+<TabItem value="serving-config" label="Serving Configuration">
 
+**Optimize MLflow serving configuration:**
 
-    @app.post("/predict")
-    async def predict(request: Request):
-        # Route traffic based on split
-        rand = random.random()
-
-        if rand < TRAFFIC_SPLIT["model_a"]:
-            endpoint = MODEL_A_URL
-            model_version = "A"
-        else:
-            endpoint = MODEL_B_URL
-            model_version = "B"
-
-        # Forward request
-        try:
-            req_json = await request.json()
-            response = requests.post(
-                endpoint,
-                headers={"Content-Type": "application/json"},
-                data=json.dumps(req_json),
-                timeout=30,
-            )
-
-            result = response.json()
-
-            # Log for analysis
-            logging.info(f"Model: {model_version}, Request: {req_json}, Response: {result}")
-
-            return JSONResponse(
-                content={"prediction": result, "model_version": model_version}
-            )
-
-        except Exception as e:
-            logging.error(f"Error with model {model_version}: {e}")
-            return JSONResponse(content={"error": str(e)}, status_code=500)
-
-
-    if __name__ == "__main__":
-        uvicorn.run(app, host="0.0.0.0", port=8080)
-    ```
-
-  </TabItem>
-  <TabItem value="multi-model-serving" label="Multi-model Serving">
-    Serve multiple models from a single endpoint for ensemble predictions or
-    model routing based on input characteristics.
-
-    ```python
-    import mlflow
-    import pandas as pd
-    from fastapi import FastAPI, Request
-    from fastapi.responses import JSONResponse
-    import uvicorn
-
-    app = FastAPI()
-
-    # Load specialized models
-    models = {
-        "fraud_detection": mlflow.pyfunc.load_model("models:/<fraud-model-id>"),
-        "recommendation": mlflow.pyfunc.load_model("models:/<recommendation-model-id>"),
-        "classification": mlflow.pyfunc.load_model("models:/<classification-model-id>"),
-    }
+- **Input Formats**: Use `dataframe_split` format for structured data to leverage pandas optimizations
+- **Batch Processing**: Send multiple samples in a single request to improve throughput
+- **Server Settings**: Configure workers and timeout settings based on model complexity
+- **Environment Variables**: Use MLflow environment variables for configuration
+
+```bash
+# Configure serving with appropriate settings
+export MLFLOW_SCORING_SERVER_REQUEST_TIMEOUT=60
+export GUNICORN_CMD_ARGS="--timeout 60 --workers 4"
+
+mlflow models serve \
+  --model-uri models:/my_model@champion \
+  --port 5000 \
+  --enable-mlserver  # For advanced serving features
```

Comment:
I'm not sure if we still want to advertise this, we didn't support mlserver for some new features